### DISCUSSION TITLE
Web3Auth + Okta Integration?

### DISCUSSION THREAD

**Discussion by gerardopar** [Web3Auth/Web3Auth#468] [Help]
Is there any guides for getting started with web3Auth + **Okta**? I do see it as a supported Custom Login Provider

**Comment by shahbaz17** [comment]
Hey @gerardopar 

Yes, you can use Okta as Federated Identity Provider with Web3Auth. As of now, there isn't a proper guide around it. But it would be very much similar to https://web3auth.io/docs/guides/firebase 

I have also added this to our list of upcoming guides.

**Reply by gerardopar** [reply]
hey @shahbaz17 thanks for the swift reply, any timeline on when docs for okta integration might be available?
